cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,35 @@
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
```

Comment:
You need to add all files which you imported in your tests

Reply:
Done